tools/packaging/packaging.strings.psd1

```diff
@@ -109,22 +109,27 @@ OsxDistributionTemplate = @'
 '@
 NuspecTemplate = @'
 <?xml version="1.0" encoding="utf-8"?>
-<package
-  xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
-  <metadata>
-    <id>{0}</id>
-    <version>{1}</version>
-    <title>PowerShellRuntime</title>
```

Comment:
Did you intent to remove the <title> element?

Reply:
According to nuget.org documentation having a description is a must, title is not mandatory. Hence, removed it.